ci/install-kind.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+set -ex
+
+mkdir -p bin
```

Comment:
There's lots of duplication now between install-kind and install-minikube. 

I don't think we need to run tests in both minikube and kind. I think we should either switch to kind or not at all. Does that make sense? What is the value of maintaining and testing multiple bootstrap environments?

Reply:
Right, that was also my idea to incrementally remove minikube once we are happy with kind.

However the unreleased kind 0.3.0 switched to the containerd CRI runtime, as suppose to using docker runtime.  I want to bring this up to see if people are comfortable when the CI runs solely on a non-typical kubernetes environment of underlying runtime.

Reply:
I'm 100% comfortable for z2jh CI to run with kind!

Reply:
I've just tried Kind locally (not using these scripts), it's really easy to get running on a Linux system. Main downside so far is I can't get NetworkPolicies to work.